docs/_documentations/mdt-eclipse-firstproject.md

```diff
@@ -1,46 +1,48 @@
 ---
 layout: docs
-title: "Codewind for Eclipse"
-description: "Codewind for Eclipse"
+title: "Creating your first Eclipse Codewind project"
+description: "Creating your first Eclipse Codewind project"
 keywords: introducing, introduction, overview, what is, tools, vscode, visual, studio, code, java, microprofile, spring, node, nodejs, node.js, javascript, Codewind for VS Code, tools, view, debug, integrate, open a shell session, toggle auto build, manually build, scope VS Code workspace, disable, enable, delete
 duration: 1 minute
 permalink: mdt-eclipse-firstproject
 type: document
 order: 2
 parent: mdt-eclipse-overview
 ---
-# Creating your first project
+# Creating your first Eclipse Codewind project
 <br/>
-To create your first project, right-click the **Local [Running]** item and select **Create New Project...**
+To create your first project:
+
+1\. Right-click the **Local [Running]** item >  select **Create New Project...**.
 
 ![](dist/images/eclipsecreateproject1.png){:width="800px"}
 
-This will display the project creation screen
+2\. Creating a new project displays the project creation screen.
```

Comment:
Here's where captions for the screenshots would come in handy. Here on line 20, we don't actually have a new step. Rather, this sentence tells the result of Step 1. Ideally, we'd put the results of the steps together with the action that produced the results, like so: `Right-click the **Local [Running]** item and select **Create New Project...**. After you create a new project, the project creation screen appears.`